### DISCUSSION TITLE
Device mappings

### DISCUSSION THREAD

**Discussion by Ryiguchi** [kingstinct/.github#2] [General]
https://docs.google.com/spreadsheets/d/1wCPYg5agxkekLHVW1_28zETHfI-ADjkP5w3sKA-yUdo/edit?usp=sharing

**Comment by robertherber** [comment]
@Ryiguchi I moved this to the repo: https://github.com/kingstinct/lifeline/blob/main/docs/device-mappings.md - makes it easier to access and also makes it easily available for AI tools :)
